internal/database/database_test.go

```diff
@@ -226,10 +226,10 @@ func TestCheckDataset(t *testing.T) {
 	r := sqlTesterHelper(t, func(mock sqlmock.Sqlmock, testDb *SQLdb) error {
 
 		expected := true
-		query := "SELECT DISTINCT dataset_id FROM local_ega_ebi.file_dataset WHERE dataset_id = \\$1"
+		query := "SELECT stable_id FROM sda.datasets WHERE stable_id = \\$1"
```

Comment:
This makes no sense.

Reply:
Sure it does! It returns a value if the dataset ID exists, otherwise empty. It's more common to do this with `EXISTS` but I'm not here to criticize people's SQL preferences ^^